src/main/java/lotto/model/RankReward.java

```diff
@@ -0,0 +1,25 @@
+package lotto.model;
+
+import java.util.List;
+
+public class RankReward {
+
+    private List<Rank> rankList;
+
+    public RankReward(List<Rank> rankList) {
+        this.rankList = rankList;
+    }
+
+    public int getWinnerCount(RewardStatus rewardStatus) {
+        return (int) rankList
+                        .stream()
+                        .filter(o -> o.getCountOfMatch() == rewardStatus.getMatchingCount()
+                                        && o.getBonus() == rewardStatus.isMatchingBonus()).count();
```

Comment:
o가 Rank이므로 바로 동등성을 비교할 수 있을 것 같습니다!
```suggestion
                        .filter(o -> o == rewardStatus).count();
```